lib/mitmproxy-addon.py

```
@@ -0,0 +1,107 @@
+            "port": flow.request.port,
+            "path": flow.request.path,
+            "headers": dict(flow.request.headers),
+            "content_length": len(flow.request.content) if flow.request.content else 0,
+        }
+        
+        # Log request body for POST/PUT/PATCH (but limit size)
+        if flow.request.method in ["POST", "PUT", "PATCH"]:
+            if flow.request.content and len(flow.request.content) < 10000:
+                try:
+                    request_data["body"] = flow.request.text
+                except:
```

Ruff (0.14.3)

Do not use bare `except`

(E722)